src/Files.App/Utils/StatusCenter/StatusCenterItemProgressModel.cs

```diff
@@ -1,19 +1,30 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
+using System.Collections.Concurrent;
+using System.Runtime.CompilerServices;
+
 namespace Files.App.Utils.StatusCenter
 {
 	/// <summary>
 	/// Represents a model for file system operation progress.
 	/// </summary>
-	public class StatusCenterItemProgressModel : ObservableObject
+	public class StatusCenterItemProgressModel : INotifyPropertyChanged
```

Comment:
Why did you changed from ObservableObject to INotifyPropertyChanged?

Reply:
See the below `SetProperty` method.

I need to use a dirty tracker and then bulk trigger the update with a debouncer, so we cannot use the `ObservableObject` which will notify the update on every change happening.

Reply:
Thank you for explanation!